logstash-core/lib/logstash/errors.rb

```diff
@@ -1,4 +1,5 @@
 # encoding: utf-8
+require "logstash-core" # for LogStash::Error
```

Comment:
it took me an unfortunate amount of digging to find that `LogStash::Error` comes from `org.logstash.RubyUtil`; maybe adding this to the comment?

~~~ diff
-require "logstash-core" # for LogStash::Error
+require "logstash-core" # for LogStash::Error from org.logstash.RubyUtil.LOGSTASH_ERROR
~~~